hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -404,10 +404,23 @@ public boolean isWorkloadProfileNeeded() {
 
 
   public Iterator<List<WriteStatus>> handleUpdate(String commitTime, String fileLoc,
-      Iterator<HoodieRecord<T>> recordItr)
+                                                  Iterator<HoodieRecord<T>> recordItr)
       throws IOException {
     // these are updates
     HoodieMergeHandle upsertHandle = getUpdateHandle(commitTime, fileLoc, recordItr);
+    return handleUpdateInternal(upsertHandle, commitTime, fileLoc);
+  }
+
+  public Iterator<List<WriteStatus>> handleUpdate(String commitTime, String fileLoc,
```

Comment:
can we think of a better way, without introducing a new method here in the table class?

Reply:
okay, let me think about it.